Various docs articles (#5)
Kubulambula/Godot-GDShell: addons/gdshell/docs/en/commands/man.md

<a href="https://github.com/Kubulambula/Godot-GDShell">
  <img src="https://github.com/Kubulambula/Godot-GDShell/blob/main/addons/gdshell/docs/assets/logo.png" align="left" width="80" height="80">
</a>

> # GDShell Command `man`

Reviewer: ```suggestion
# `man`
```

I think that the `GDShell Command` part is not necesarry

> [!alert]
> Not all arguments and flags in this docs article are currently implemented. 

> The `man` command is used to retrive information about the terminal and its commands. 

Reviewer: ```suggestion
The `man` command is used to retrieve information and manuals for commands. 
```

Better wording.

What do you mean by "information about the terminal"?

Syntax: ``man (command)``

## (command)

You can specify a command to list the manual article for, for example `man gdfetch` will print the manual for the `gdfetch` command. If no command is specified, it will default to the manual article for the manual (being the same as `man man`). 

> The manual command simply calls the passed commands `_get_manual()` function. 

Reviewer: ```suggestion
The manual command gets the manual by calling the passed commands `_get_manual()` function. See [Custom commands](custom_commands.md#command-manual)
```

Better wording + tutorial link
